A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -492,23 +492,87 @@ async function loadFuel(mcdu, updateView) {
     const innerTankCapacity = 1816; // Left and Right // Value from flight_model.cfg
     const centerTankCapacity = 2179; // Center // Value from flight_model.cfg
 
-    const fuelWeightPerGallon = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "kilograms");
-    let currentBlockFuelInGallons = +currentBlockFuel / +fuelWeightPerGallon;
-
-    const outerTankFill = Math.min(outerTankCapacity, currentBlockFuelInGallons / 2);
-    await SimVar.SetSimVarValue(`FUEL TANK LEFT AUX QUANTITY`, "Gallons", outerTankFill);
-    await SimVar.SetSimVarValue(`FUEL TANK RIGHT AUX QUANTITY`, "Gallons", outerTankFill);
-    currentBlockFuelInGallons -= outerTankFill * 2;
-
-    const innerTankFill = Math.min(innerTankCapacity, currentBlockFuelInGallons / 2);
-    await SimVar.SetSimVarValue(`FUEL TANK LEFT MAIN QUANTITY`, "Gallons", innerTankFill);
-    await SimVar.SetSimVarValue(`FUEL TANK RIGHT MAIN QUANTITY`, "Gallons", innerTankFill);
-    currentBlockFuelInGallons -= innerTankFill * 2;
-
-    const centerTankFill = Math.min(centerTankCapacity, currentBlockFuelInGallons);
-    await SimVar.SetSimVarValue(`FUEL TANK CENTER QUANTITY`, "Gallons", centerTankFill);
-    currentBlockFuelInGallons -= centerTankFill;
-
+    const fuelWeightPerGallon = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "kilograms"); // current value
+    //Aux values
+    const fuelLAux = SimVar.GetSimVarValue(`FUEL TANK LEFT AUX QUANTITY`, "Gallons");
+    const fuelRAux = SimVar.GetSimVarValue(`FUEL TANK RIGHT AUX QUANTITY`, "Gallons");
+    //Main Values
+    const fuelLMain = SimVar.GetSimVarValue(`FUEL TANK LEFT MAIN QUANTITY`, "Gallons");
+    const fuelRMain = SimVar.GetSimVarValue(`FUEL TANK RIGHT MAIN QUANTITY`, "Gallons");
+    //Center values
+    const fuelCenter = SimVar.GetSimVarValue(`FUEL TANK CENTER QUANTITY`, "Gallons");
+    const currentBlockFuelInGallons = +currentBlockFuel / +fuelWeightPerGallon;
+    const fuelcount = 0;
+    if (fuelWeightPerGallon != currentBlockFuelInGallons) {
+        const startFuel = setInterval(function () {
```

Comment:
**question**: Why should this code execute every 5 milliseconds?

Reply:
this should also be using the `() => {` syntax instead of using `function () {`